.changeset/silent-jars-tell.md

```diff
@@ -0,0 +1,5 @@
+---
+"@directus/api": patch
+---
+
+Implemented schema sanitization for GraphQL throwing an error for invalid collections or relations
```

Comment:
Something like this seems a bit more accurate to the latest version of this PR

```suggestion
Implemented schema sanitization for GraphQL, ignoring collections/relations with incompatible names instead of blocking GraphQL queries
```